### DISCUSSION TITLE
how delete this?

### DISCUSSION THREAD

**Discussion by 0ul0** [Heroic-Games-Launcher/HeroicGamesLauncher#1121] [Q&A] [answered]
I can't delete this launcher. help me please.

**Comment by tech189** [comment, marked as answer]
If you're using Windows you can uninstall it from Apps & Features in Settings. Or with `winget uninstall HeroicGamesLauncher.HeroicGamesLauncher`. On macOS, delete it from the Applications folder. On Linux, use the uninstall/remove command from your package manager.
